keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -761,13 +808,26 @@ API-initiated eviction or taint-based eviction).
 - `ResourceLimitExceeded`: indicates the pod is evicted due to exceeding
 one of it's configured limits (e.g. on memory or ephemeral-storage).
 
-The more detailed information about the reason of disruption or the exceeded
-resource limit is conveyed by the `reason` and `message` fields of the added pod
-condition.
+A more detailed information about the condition (containing the kubernetes
+component name which initiated the disruption or the resource name for the
+exceeded resource limit) is conveyed by the `reason` and `message` fields of
+the added pod condition.
 
 Finally, we are going to cover the handling of pod failures associated with the
 new pod condition types in integration tests.
 
+#### Stale DisruptionTarget condition which is not cleaned up
+
+It is possible that a stale disruption condition
+(see: [Failing delete after a condition is added](#failing-delete-after-a-condition-is-added))
+is not clean up by the disruption controller if the Pod completes. Further, if the
```

Comment:
is not cleaned up before a Pod terminates for a different reason.

Reply:
Rephrased.